includes/core/classes/class-block.php

```diff
@@ -72,4 +73,73 @@ public function register_blocks(): void {
 			);
 		}
 	}
+
+
+	/**
+	 * Register block patterns.
+	 *
+	 * This method registers multiple different block-patterns for GatherPress.
+	 *
+	 * @since 1.0.0
+	 * @see   https://developer.wordpress.org/reference/functions/register_block_pattern/
+	 *
+	 * @return void
+	 */
+	public function register_block_patterns(): void {
+
+		/**
+		 * Made to be used with the 'template' parameter
+		 * when registering the 'gatherpress_event' post type
+		 * and will not be visible to the editor at any point.
+		 */
+		\register_block_pattern(
```

Comment:
Don't need \

Reply:
Will be fixed, but I'm wondering why the CS checks didn't jump in here. Some days ago, I remember that phpcs was jelling at me because of a \WP_Query. But here, nothing. Hmmm.